cddl_gen/cddl_gen.py

```diff
@@ -2379,9 +2386,15 @@ def process_code(args):
     verbose_print(args.verbose, "Parsed CDDL types:")
     verbose_pprint(args.verbose, my_types)
 
+    git_sha = ''
+    if args.git_sha_header:
+        git_args = ['git', 'rev-parse', '--verify', '--short', 'HEAD']
+        git_sha = Popen(git_args, cwd=P_REPO_ROOT, stdout=PIPE).communicate()[0].decode('utf-8').strip()
```

Comment:
Have you verified this on windows? I just see PIPE and I fear that it won't work.

Reply:
Windows testing will come later :). PIPE here is a subprocess-specific thing, and just means that the output will be available in the Python code instead of being sent to the terminal.